src/pixelator/analysis/polarization/__init__.py

```diff
@@ -1,17 +1,15 @@
-"""
-Copyright (c) 2023 Pixelgen Technologies AB.
-"""
+"""Copyright (c) 2023 Pixelgen Technologies AB."""
+
 
 import warnings
 
+import esda.moran
 from esda.moran import Moran
 
 from pixelator.graph.utils import Graph, create_node_markers_counts
 from pixelator.statistics import (
-    binarize_counts,
```

Comment:
There are several places where we refer to `binarize_counts` and `denoise` in pixelator. Also, places where we are testing those functionalities. If we get rid of these functions, let's just delete them.

Reply:
I removed binarize now, but kept `denoise` as it is referred to in many more places. Feels like it would take a more concerted effort to get rid of it completely.

Reply:
I'll make a ticket for this and we can discuss together if we should remove it or not